install/vikunja-install.sh

```diff
@@ -26,7 +26,7 @@ cd /opt
 RELEASE=$(curl -s https://dl.vikunja.io/vikunja/ | grep -oP 'href="/vikunja/\K[0-9]+\.[0-9]+\.[0-9]+' | sort -V | tail -n 1)
 wget -q "https://dl.vikunja.io/vikunja/$RELEASE/vikunja-$RELEASE-amd64.deb"
 $STD dpkg -i vikunja-$RELEASE-amd64.deb
-sudo sed -i 's/^  timezone: .*/  timezone: UTC/' /etc/vikunja/config.yml
+sed -i 's|^  timezone: .*|  timezone: UTC|; s|"./vikunja.db"|"/etc/vikunja/vikunja.db"|; s|./files|/etc/vikunja/files|' /etc/vikunja/config.yml
```

Comment:
split to three lines for readability

Reply:
I did not mean split the string to three lines, please see other scripts how `sed` is commonly used. You can use `-e` for multiple expressions or separate `sed` commands. More than 2 expressions -> split to separate lines

Reply:
Please let me know if the new change is sufficient.

Reply:
i think @havardthom means like the tandoor-script (copy only)

```bash
sed -i -e "s|SECRET_KEY=.*|SECRET_KEY=$secret_key|g" \
       -e "s|POSTGRES_HOST=.*|POSTGRES_HOST=localhost|g" \
       -e "s|POSTGRES_PASSWORD=.*|POSTGRES_PASSWORD=$DB_PASS|g" \
       -e "s|POSTGRES_DB=.*|POSTGRES_DB=$DB_NAME|g" \
       -e "s|POSTGRES_USER=.*|POSTGRES_USER=$DB_USER|g" \
       -e "\$a\STATIC_URL=/staticfiles/" /opt/tandoor/.env
``` 

https://github.com/community-scripts/ProxmoxVE/blob/a4dd7f5449f4b1049b9a4bec2d8aec20286aab09/install/tandoor-install.sh#L68-L75